src/sssom/io.py

```diff
@@ -130,6 +130,23 @@ def split_file(input_path: str, output_directory: Union[str, Path]) -> None:
     write_tables(splitted, output_directory)
 
 
+def convert_plain_prefix_map_to_extended(prefix_map):
```

Comment:
## Refactoring from Charlie's `curies` update
I believe that https://github.com/cthoyt/curies/pull/99 will address some/all of the same changes here, so, assuming we end up merging this PR, will probably need to refactor afterwards to utilize the `curies` solution instead.

Reply:
done in 57dd171